WISE-Community/WISE
# Fixed node icon not updating when navigating to a different node

To test:
1. Open a unit in preview mode that has at least one or two steps with node icons set.
1. Navigate to different steps. Verify that the node icon changes when switching steps.
1. Open the unit in the authoring tool.
1. Open a step for editing.
1. Navigate to different steps. Verify that the node icon changes when switching steps.
1. Open a run unit in the classroom monitor that has at least one or two steps with node icons set.
1. Open a step in the Grade By Step view.
1. Navigate to different steps. Verify that the node icon changes when switching steps.

(Also updated styles to fix alignment of the node icon.)

Resolves #2200.
